zip-reduce-shielded_tx_fee.rst

```diff
@@ -0,0 +1,157 @@
+::
+
+  ZIP: 313
+  Title: Reduce default Shielded Transaction fee to 1000 zats
+  Owners: Aditya Bharadwaj <nighthawk24@gmail.com>
+  Original-Author: Aditya Bharadwaj
+  Status: Active
+  Community Status: Request for comments : https://forum.zcashcommunity.com/t/zip-reduce-default-shielded-transaction-fee-to-1000-zats/37566
+  Category: Consensus
+  Created: 2020-10-11
+  License: MIT
+
+
+Terminology
+===========
+
+The key words "MUST", "SHOULD", "SHOULD NOT", "MAY", "RECOMMENDED",
+"OPTIONAL", and "REQUIRED" in this document are to be interpreted as
+described in RFC 2119. [#RFC2119]_
+
+"Z" refers to shielded address.
+"T" refers to transparent address.
+
+The term "default transaction fee" or "conventional transaction fee"
+in this document is in reference to Z to Z transaction fee paid
+to miners on the Zcash network for the work on including
+the shielded transaction in a block.
+
+
+Abstract
+========
+
+The goal of this ZIP is to gather wallet developers, miners & Zcash users
+for consensus on reducing the default transaction fees and
+to get the Zcash default transaction fee reduced from 10,000 zats
+to 1000 zats.
+
+With reduced fees, it will be cheaper to transact on the Zcash network,
+while also inviting novel use cases for privacy
+preserving applications that would benefit from the Privacy,
+Security and Programmable Money aspects of the Zcash chain.
+
+
+Out of Scope for this Proposal
+============
+
+Discussion around transaction fees for T to T, T to Z or Z to T.
+
+
+Motivation
+============
+
+The default transaction fee presently is 0.0001 ZEC or 10,000 zats.
+With ZEC market price of $100, a user can send 10,000 transactions
+for 1 ZEC, that turns out to 1 cent per transaction and it rises
+with the increase in the price of ZEC.
+
+With increase in light wallet adoptions on mobile clients, many users
+will be new to the Zcash eco-system. And the fact that the
+transaction fees are paid by the sender (not the receiver) is
+new information to users who might use Zcash for e-commerce
+and app interactions that might result in several transactions each day.
+
+Privacy must not cost a premium. The cost of 10 shielded transactions
+buys 1GB of mobile data `in India today <https://www.cable.co.uk/mobiles/worldwide-data-pricing/>`.
+
+Zcash users must be able to do more with their ZEC balance
+than worry about paying the premium for shielded transactions.
+
+
+Requirements for consensus
+-------------------------
+
+The change to the conventional transaction fees must be undertaken sooner
+as it gets difficult to gain consensus with the growth in the network
+of wallets, exchanges, miners and third parties involved.
+
+The following parties need to be part of the consensus:
+
+* A guarantee from mining groups is required to include the lowered default fee transactions in the next block.
```

Comment:
This doesn't need to be a guarantee. Inclusion of transactions is never guaranteed. There just needs to be a reasonable expectation that transactions will be mined fairly promptly (not necessarily in the next block).